Makefile

```diff
@@ -4,27 +4,21 @@ exe = github.com/aelsabbahy/goss/cmd/goss
 pkgs = $(shell ./novendor.sh)
 cmd = goss
 TRAVIS_TAG ?= "0.0.0"
-GO_FILES = $(shell find . \( -path ./vendor -o -name '_test.go' \) -prune -o -name '*.go' -print)
 GO111MODULE=on
 
 .PHONY: all build install test release bench fmt lint vet test-int-all gen centos7 wheezy precise alpine3 arch test-int32 centos7-32 wheezy-32 precise-32 alpine3-32 arch-32
 
-all: test-all dgoss-sha256
+all: test-short-all test-int-all dgoss-sha256
 
-test-all: fmt lint vet test test-int-all
+test-short-all: fmt lint vet test
 
 install: release/goss-linux-amd64
 	$(info INFO: Starting build $@)
 	cp release/$(cmd)-linux-amd64 $(GOPATH)/bin/goss
 
 test:
 	$(info INFO: Starting build $@)
-	{ \
```

Comment:
Similar to inside travis, factored these into scripts where it becomes easier to deal with the cross-platform differences.